cristin-project/src/test/java/no/unit/nva/cristin/projects/query/projectQuery.feature

```diff
@@ -123,16 +124,16 @@ Feature: API tests for Cristin projects query
       | 'page'      | 'hello'             |
       | 'results'   | 'hello'             |
 
-  Scenario: Query with missing query parameter returns Bad Request
+  Scenario: Query with missing query parameter do returns results
```

Comment:
So now we allow doing a search without specifying any parameters

Reply:
you need **one** query param, but anyone goes?

Reply:
I think we can just call the endpoint without any params, because this uri to cristin is also valid:
https://api.cristin-test.uio.no/v2/projects

I think we should allow doing a query without specifying params as you have done here. I asked because i just wanted to know if we could or not. It was not a request to change it

Reply:
This was the main reason to do all this work , I needed one optional parameter to be filled in in order to validate query